ecosystem/theme-default/src/client/components/Page.vue

```diff
@@ -1,6 +1,13 @@
 <script setup lang="ts">
 import PageMeta from '@theme/PageMeta.vue'
 import PageNav from '@theme/PageNav.vue'
+
+defineSlots<{
+  'top'?: (props: Record<never, never>) => any
```

Comment:
Any way to avoid those verbose `Record<never, never>` and `any` ?

Reply:
Currently volar complains if props are not provided, the any is expected.

See https://blog.vuejs.org/posts/vue-3-3#typed-slots-with-defineslots and https://github.com/vuejs/core/issues/7982